adapters/smartx/smartx.go

```diff
@@ -0,0 +1,116 @@
+package smartx
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+// adapter is a implementation of the adapters.Bidder interface.
+type adapter struct {
+	endpointURL string
+}
+
+func Builder(_ openrtb_ext.BidderName, config config.Adapter, _ config.Server) (adapters.Bidder, error) {
+	//println("Builder smartx", config.Endpoint)
+	// initialize the adapter and return it
+	return &adapter{
+		endpointURL: config.Endpoint,
+	}, nil
+}
+
+// MakeRequests prepares the HTTP requests which should be made to fetch bids.
+func (a *adapter) MakeRequests(openRTBRequest *openrtb2.BidRequest, requestInfo *adapters.ExtraRequestInfo) (requestsToBidder []*adapters.RequestData, errs []error) {
+	// parse the requests answer
+	openRTBRequestJSON, err := json.MarshalIndent(openRTBRequest, "", "   ")
```

Comment:
Is there a reason to choose MarshalIndent instead of MarshalJson? I think - adding an indentation would slightly take performance hit. Requesting you to please use MarshalJson if possible.

Reply:
thank you, debug leftover. adjusted